tests/unit/s2n_x509_validator_time_verification_test.c

```diff
@@ -13,12 +13,10 @@
 * permissions and limitations under the License.
 */
 
+#include "crypto/s2n_libcrypto.h"
 #include "s2n_test.h"
 #include "testlib/s2n_testlib.h"
 
-bool s2n_libcrypto_supports_flag_no_check_time();
-uint64_t s2n_libcrypto_awslc_api_version(void);
```

Comment:
It's not, this change is a cleanup that probably should have been included in [the previous PR](https://github.com/aws/s2n-tls/pull/4823/files). This is the only other use of `s2n_libcrypto_awslc_api_version()` in the s2n unit tests. Now that this API was added to `s2n_libcrypto.h` I updated this test to include the header instead of declaring the API itself.